gallery/src/components/NxTable/NxTableFilterExample.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTable,
+  NxTableBody,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow,
+  NxFilterInput
+} from '@sonatype/react-shared-components';
+import { toLower } from 'ramda';
+
+const tableData = [
+  {name: 'Anna', country: 'USA'},
+  {name: 'Lean', country: 'France'},
+  {name: 'Louis', country: 'France'},
+  {name: 'Zach', country: 'Colombia'},
+  {name: 'Jimmy', country: 'Germany'}
+];
+
+interface Row { name: string; country: string };
+
+const NxTableFilterExample = () => {
+
+  const [rows, setRows] = useState(tableData);
+  const [nameFilter, setNameFilter] = useState('');
+  const [countryFilter, setCountryFilter] = useState('');
+
+  const listId = 'countryList';
+  const countries = new Set(tableData.map((row: Row) => {
+    return row.country;
+  }).sort());
+
+  const applyFilter = (rows: Row[], name: string, country: string) => {
+    let filteredRows = rows;
+    if (name !== '') {
+      filteredRows = filteredRows.filter(
+          (row: Row) => toLower(row.name).includes(toLower(name))
+      );
+    }
+    if (country !== '') {
+      filteredRows = filteredRows.filter(
+          (row: Row) => toLower(row.country).includes(toLower(country))
+      );
+    }
+    return filteredRows;
+  };
+
+  const onFilterNameChange = (filter: string) => {
+    setNameFilter(filter);
+    setRows(applyFilter(tableData, filter, countryFilter));
+  };
+
+  const onFilterCountryChange = (filter: string) => {
+    setCountryFilter(filter);
+    setRows(applyFilter(tableData, nameFilter, filter));
+  };
+
+  return (
+    <NxTable>
+      <NxTableHead>
+        <NxTableRow>
+          <NxTableCell>Name</NxTableCell>
+          <NxTableCell>Country</NxTableCell>
+        </NxTableRow>
+        <NxTableRow>
+          <NxTableCell isFilter>
+            <NxFilterInput placeholder="Type a name"
+                           onChange={onFilterNameChange}
+                           value={nameFilter}/>
+          </NxTableCell>
+          <NxTableCell isFilter>
+            <NxFilterInput placeholder="Select a country"
+                           list={listId}
+                           onChange={onFilterCountryChange}
+                           value={countryFilter}/>
+            <datalist id={listId}>
+              {
+                Array.from(countries).map((country: string) => (
```

Comment:
`Array.from` doesn't work in IE.  Rather than dealing with conversion to/from a `Set`, I'd use ramda's `uniq`

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/967b115b69ecda00ff623667bb9a2ce7d38ef1d1